extensions/amazon-aws/src/util/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { AWS_URL_BASE } from "../constants";
```

Comment:
Is this file added by mistake? We already have https://github.com/raycast/extensions/blob/main/extensions/amazon-aws/src/util/index.ts and providing this file will confuse TS, which is why you're getting the build error. It is a good idea to run `npm run build` before pushing a commit.

Reply:
hm, this is the result of the automatic merge, which i triggered with
```
npx @raycast/api@latest pull-contributions
```